subsys/modem/Kconfig

```diff
@@ -31,8 +31,19 @@ if MODEM_CMUX
 
 config MODEM_CMUX_WORK_BUFFER_SIZE
 	int "CMUX module work buffer size in bytes"
+	range 23 1507
+	default 38
+	help
+	  Size of the work buffer used by the CMUX module.
+	  Recommended size is MODEM_CMUX_MTU + 7 (CMUX header size).
```

Comment:
Should there be a build assert that verifies that this size is larger than the cmux mtu?

Reply:
I did not test this, but my understanding is that both CMUX and PPP are able to work even if the input buffer size is only one byte. It would be just so unoptimal that data drops would break the packets.

Therefore I would not like to add asserts, until I really know that lower values break things.